samples/icons-pack/src/components/IconList/IconListModel.ts

```diff
@@ -0,0 +1,13 @@
+import { ComponentModelBase, serializable } from "@vertigis/web/models";
+import { FrameworkServiceType, inject } from "@vertigis/web/services";
+import type { IconRegistry } from "@vertigis/web/ui/icons";
+
+@serializable
+export default class IconListModel extends ComponentModelBase {
+    @inject(FrameworkServiceType.ICON_REGISTRY)
+    imageRegistry: IconRegistry;
```

Comment:
It's a bit odd to rename it to "imageRegistry" here when it's called an icon registry. I suspect this is some copy/paste leftover from its old name in Web.